app/app.go

```diff
@@ -0,0 +1,26 @@
+package app
+
+import (
+	"encoding/json"
+	"net/http"
+
+	controllers "github.com/EloYaniel/academy-go-q42021/controllers"
+	"github.com/gorilla/mux"
+)
+
+func InitApp() *mux.Router {
+	r := mux.NewRouter()
+
+	r.HandleFunc("/health", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
It wouldn't be a bad idea to keep this function and the next one(the handler for the `/` route) in a separate file. 
At least for the `/health` route you could eventually do more checks to determine if the service is or not healthy